src/main/resources/log4j2-spring.xml

```diff
@@ -10,6 +10,7 @@
         </Console>
     </Appenders>
     <Loggers>
+        <Logger name="org.zalando.logbook.Logbook" level="TRACE"/>
```

Comment:
should logbook always be enabled? I'm worried about the vast amount of log data and if our logging solution is able to handle it.